src/services/codefixes/importFixes.ts

```diff
@@ -621,6 +618,27 @@ namespace ts.codefix {
         }
     }
 
+    /**
+     * This function should be used to insert nodes alphabetically in lists when nodes don't carry separators as the part of the node range,
+     * e.g. export lists, import lists, etc.
+     * Linear search is used instead of binary as the (generally few) nodes are not guaranteed to be in order.
+     */
+    function insertNodeInsideListNodesAlphabetically(context: SymbolContext, sourceFile: SourceFile, containingList: NodeArray<ImportSpecifier>, newNode: ImportSpecifier) {
+        return ChangeTracker.with(context, t => {
+            if (compareStringsCaseInsensitive(newNode.name.text, containingList[0].name.text) === Comparison.LessThan) {

```

Comment:
I'd be happier if this were more tightly coupled to organizeImports so that they don't get out of sync.  Maybe we could export `compareIdentifiers` from `organizeImports.ts` and use it here?